src/Codeception/Module/Symfony2.php

```diff
@@ -382,7 +391,7 @@ public function seeInCurrentRoute($routeName)
         }
 
         try {
-            $matchedRouteName = $router->match($this->grabFromCurrentUrl())['_route'];
+            $matchedRouteName = $router->match(explode('?', $this->grabFromCurrentUrl())[0])['_route'];
         } catch (\Exception\ResourceNotFoundException $e) {
```

Comment:
Not really, above we need ALL the parameters to do the comparison, at this method we only need '_route' because we will ignore parameters.